pandas/core/nanops.py

```diff
@@ -879,31 +882,41 @@ def reduction(
 
 @disallow("O")
 def nanargmax(
-    values: np.ndarray,
+    values: Union[np.ndarray, "Series"],
```

Comment:
IIRC a couple weeks @jorisvandenbossche was saying funcs in this module shouldnt accept Series.  I'd expect this to be ArrayLike

Reply:
I will check, but if we don't currently pass a series, could just remove (or not add). I added it because the example in the docstring passed a Series.

Reply:
> I'd expect this to be ArrayLike

pandas\core\nanops.py:924: error: "ExtensionArray" has no attribute "argmax"
pandas\core\nanops.py:970: error: "ExtensionArray" has no attribute "argmin"

will just revert to ndarray for now and change example in docstring to use numpy array.